src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.logo img {
+  margin: 0;
+  padding: 0;
+  width: 40px;
+  height: 40px;
+}
+
+ul.list {
+  margin: 0;
+  padding: 0;
+}
+
+.list {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.list_li {
+  position: relative;
+  margin-right: 20px;
+}
+
+.list_nav.is-active::after {
+  position: absolute;
+  content: "";
+  left: 0;
+  bottom: 0;
+  background-color: #00acdc;
+  height: 4px;
+  width: 100%;
+  border-radius: 4px;
+}
+
+.list_li:last-child {
+  margin-right: 0;
+}
```

Comment:
to avoid 

```
.list_li {
  margin-right: 20px;
}

.list_li:last-child {
  margin-right: 0;
}
```

you can just use
```
.list_li:not(:last-child) {
  margin-right: 20px;
}
```